### DISCUSSION TITLE
Úprava menu + hlavní stránky

### DISCUSSION THREAD

**Discussion by JarkaP** [Knihovna-AVCR-web/historicka-korespondence-web#7] [General]
Menu je upravené podle návrhu v #4, projekty pak přesunuté do samostatného oddílu, viz https://historicka-korespondence.cz/v2/

**Comment by TerezaMad** [comment]
Ahoj Jarko,
chci se zeptat na projekty, které jsou v návrhu rozřazeny do dlaždic – jak dlouhý text je tam možné vkládat? Asi jako je to v návrhu u Masaryka? Nebo se dlaždice "roztahují" spolu s textem. Jde mi o to, aby to pak nějak vypadalo a nebyla každá dlaždice jinak dlouhá, ale možná to moc řeším a příp, si s textem nějak pohraji. :-)

**Reply by JarkaP** [reply]
Pevnou výšku to nemá, výška všech dlaždic v řádku se přizpůsobí té nejvyšší.
